### PR TITLE
Chicago file 

### LINKED ISSUE #51: FakeCode - HotFix
- [x] Create a branch
- [ ] Fix & Commit

### PR DESCRIPTION
**Pull request by FakeCode**
This pull requests resolves #51 
added things to do

@beardofedu Can you review my changes:question:  :eyes:

### CODE REVIEW

File: FakeCode-Chicago.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Chicago
+
+### Things to do
+-- still am thinking
```
Comment (FakeCode) [inline FakeCode-Chicago.md]: its time to stop thinking

**Comment by prkos** [comment]
Never surrender! :rocket: Only take breaks :pizza: :beer:

File: FakeCode-Chicago.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+-- any idea! :eyes
```
Comment (prkos) [inline FakeCode-Chicago.md]: I suggest meditation :eyes:

File: FakeCode-Chicago.md (hunk 1 above)
Comment (deatonjm) [inline FakeCode-Chicago.md]: @FakeCode Lollapalooza is pretty good if you can stand the crowds :musical_keyboard: :guitar: :musical_note:

**Comment by FakeCode** [comment]
All is well! :shipit:

**Pull request merged**